src/plugins/plugin.tooltip.js

```diff
@@ -1004,6 +1004,7 @@ export class Tooltip extends Element {
     if (changed || positionChanged) {
       this._active = active;
       this._eventPosition = eventPosition;
+      this._ignoreOneReplay = true;
```

Comment:
Got a bit confused by the name. I thought it would block only the first replay, but it seems to block all replays until a non replay event happens

Reply:
Right, I changed the behavior after naming the variable. Any suggestions?
I can think of some very long ones, but I don't like those :)

Reply:
maybe just `_ignoreReplayEvents`?